src/mobility/components/OperatorBenefit.tsx

```diff
@@ -56,15 +59,70 @@ export const OperatorBenefit = ({
   );
 };
 
+const BenefitImageAsset = ({formFactor} : {formFactor: FormFactor}) => {
+  switch (formFactor) {
+    case FormFactor.Car:
+      return <BundlingCarSharing />;
+    case FormFactor.Bicycle:
+      return <BundlingCityBike />;
+    default:
+      return <View />;
+  }
+}
+
+type BenefitImageProps = {
+  formFactor: FormFactor;
+  eligible: boolean;
+  style?: ViewStyle;
+};
+
+const BenefitImage = ({
+  formFactor,
+  eligible,
+  style,
+}: BenefitImageProps): JSX.Element => {
+
+  return (
+    <View style={style}>
+      <>
+        <BenefitImageAsset formFactor={formFactor} />
+        {eligible && (<BenfitEligibilityIndicator />)}
+      </>
+    </View>
+  );
+};
+
+const BenfitEligibilityIndicator = () => {
+  const styles = useStyles();
+
+  return (
+    <View style={styles.indicator}>
+      <Check width={24} height={24} />
+    </View>
+  );
+};
+
+
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   benefitContainer: {
     flexDirection: 'row',
+    alignItems: 'center',
+  },
+  benefitContent: {
+    flex: 4,
+    marginStart: theme.spacings.medium,
   },
   benefitImage: {
```

Comment:
Not in use

Reply:
it is now used to add margin to the image